src/test/java/net/snowflake/client/jdbc/DatabaseMetaDataInternalIT.java

```diff
@@ -323,9 +323,12 @@ public void testGetFunctionColumns() throws SQLException
     assertEquals("FUNC112() RETURN TABLE (COLA VARCHAR, COLB NUMBER, BIN2 BINARY, SHAREDCOL NUMBER)",
                  resultSet.getString("SPECIFIC_NAME"));
     assertFalse(resultSet.next());
+    resultSet = databaseMetaData.getFunctionColumns(null, "%", "%", "%");
+    // we have 81 columns returned
+    assertEquals(81, getSizeOfResultSet(resultSet));
     resultSet = databaseMetaData.getFunctionColumns("%", "%", "%", "%");
-    assertFalse(resultSet.next());
-
+    // we have 81 columns returned
```

Comment:
Isn't this 81? or 0?

Reply:
Does this cover the fixed case?

Reply:
Oh I copied and pasted the same comment from the assertion above; that's an error. Should be 0.

Reply:
ok, so all columns in all databases are zero?

Reply:
Will fix the comment

Reply:
Right, using "%" does not work as a wildcard for catalog (only for schema, function name, function pattern, etc). This applies for all get functions (getTables(), getColumns(), etc) and has been recently documented. So the test is included to reflect that. Updated the comment to be accurate.